### PR TITLE
Rename DataflowExampleUtils and DataflowExampleOptions

### PR DESCRIPTION
**Pull request by peihe**

### CODE REVIEW

**Comment by peihe** [comment]
R: @dhalperi

File: examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java [hunk 1]
```diff
@@ -195,7 +192,7 @@ public static void main(String[] args) throws IOException {
     options.setBigQuerySchema(getSchema());
     // DataflowExampleUtils creates the necessary input sources to simplify execution of this
     // Pipeline.
-    DataflowExampleUtils exampleDataflowUtils = new DataflowExampleUtils(options,
+    ExampleUtils exampleDataflowUtils = new ExampleUtils(options,
       options.isUnbounded());
```
Comment (dhalperi) [inline examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java]: can we fix any wrapping in this file?

File: examples/java/src/main/java/org/apache/beam/examples/complete/TrafficRoutes.java [hunk 2]
```diff
@@ -317,7 +317,7 @@ public ReadFileAndExtractTimestamps(String inputFile) {
   *
   * <p>Inherits standard configuration options.
   */
-  private interface TrafficRoutesOptions extends DataflowExampleOptions,
+  private interface TrafficRoutesOptions extends ExampleOptions,
```
Comment (dhalperi) [inline examples/java/src/main/java/org/apache/beam/examples/complete/TrafficRoutes.java]: similar wrap question?

**Comment by dhalperi** [comment]
LGTM

File: examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java (hunk 1 above)
Comment (peihe) [inline examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java]: done

File: examples/java/src/main/java/org/apache/beam/examples/complete/TrafficRoutes.java (hunk 2 above)
Comment (peihe) [inline examples/java/src/main/java/org/apache/beam/examples/complete/TrafficRoutes.java]: done

**Comment by peihe** [comment]
PTAL

**Comment by dhalperi** [comment]
LGTM

**Pull request closed without merging**